asteroid/dsp/declipp.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+
+def declipp(mix_np, est_np):
+    """
+
+    Args:
+        mix_np (numpy array): One mixture
+        est_np (numpy array): Estimates (nb_estimates, time)
+
+    """
+    for i in range(len(est_np)):
+        est_np[i] *= np.max(np.abs(mix_np)) / np.max(np.abs(est_np[i]))
+    return est_np
```

Comment:
- This modifies the array in-place, which is a bad idea. 
- I'd use list comp here 
```python
mix_max = np.max(np.abs(mix_np)
return np.stack([est * mix_max / np.max(np.abs(est)) for est in est_np], dim=0)
```
- This needs a test.